internal/entities/properties/service/service.go

```diff
@@ -79,7 +79,7 @@ func WithEntityTimeout(timeout time.Duration) propertiesServiceOption {
 
 // NewPropertiesService creates a new properties service
 func NewPropertiesService(
-	store db.Store,
+	store db.ExtendQuerier,
```

Comment:
This is correct, but reading this change made me realize that I should have exposed the ExtendQuerier as a parameter to all the property service interface methods so that we can always pass a transaction

Reply:
I'm not sure that's needed. We can pass a transaction that's already begun, and handle the rollback and commit outside of the property service.